CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ Current integrators:
     - Email: nicolas.harrand@gmail.com
 - Martin Monperrus [@monperrus](https://github.com/monperrus/)
     - Email: martin.monperrus@gnieh.org
-    - GPG fingerprint: [074F73B36D8DD649B132BAC18035014A2B7BFA92](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x074F73B36D8DD649B132BAC18035014A2B7BFA92)
+    - GPG fingerprint: [AF7B251DA8126C30896FAFF77D398AD45AEEEC93](https://keyserver.ubuntu.com/pks/lookup?op=get&search=AF7B251DA8126C30896FAFF77D398AD45AEEEC93)
```

Comment:
That's a 404 for me: https://keyserver.ubuntu.com/pks/lookup?op=get&search=AF7B251DA8126C30896FAFF77D398AD45AEEEC93?

Reply:
+0x :)